decidim-dev/lib/decidim/dev/test/factories.rb

```diff
@@ -4,12 +4,16 @@
 
 FactoryBot.define do
   factory :dummy_component, parent: :component do
+    transient do
+      skip_injection { false }
```

Comment:
Related to the renaming that I was mentioning, IMO this should be renamed to `skip_xss_injection` 

Doing a review or some code that says `skip_xss_injection: true` it's a 🚩🚩, but just seeing  `skip_injection: true` doesn't ring any bell IMO. 

(To be done after all is merged and solved if you agree)

Reply:
I do agree with the renaming, after all the work in scope is performed. ( all the upcoming PRS )